src/main/kotlin/app/revanced/patches/youtube/layout/playerbuttonbg/PlayerButtonBackgroundPatch.kt

```diff
@@ -10,7 +10,7 @@ import org.w3c.dom.Element
 @Suppress("Deprecation", "unused")
 object ForcePlayerButtonBackgroundPatch : BaseResourcePatch(
     name = "Force hide player buttons background",
-    description = "Hide the dark background surrounding the video player controls at compile time.",
+    description = "Removes the dark background surrounding the video player controls at compile time.",
```

Comment:
Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the dark background surrounding the video player controls.",
```